### PR TITLE
PTL-10823_tolerance-policies

### PR DESCRIPTION
**Pull request by rruizher**
Related: https://github.com/RedHatTraining/DO378/pull/1200

### CODE REVIEW

File: tolerance-policies/apps/monitor/src/test/java/com/redhat/training/NativeInfoResourceIT.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package com.redhat.training;
 
-import io.quarkus.test.junit.NativeImageTest;
-
+import io.quarkus.test.junit5.NativeImageTest
 @NativeImageTest
```
Comment (someth2say) [inline tolerance-policies/apps/monitor/src/test/java/com/redhat/training/NativeInfoResourceIT.java]: `@NativeImageTest` is not available anymore.
Use `@QuarkusIntegrationTest`

Comment (someth2say) [reply tolerance-policies/apps/monitor/src/test/java/com/redhat/training/NativeInfoResourceIT.java]: Also, apply this same fix to the solutions folder.

File: tolerance-policies/apps/monitor/src/main/java/com/redhat/training/MonitorResource.java
Comment (someth2say) [inline tolerance-policies/apps/monitor/src/main/java/com/redhat/training/MonitorResource.java:1]: Apply same changes to solutions

**Review by someth2say — CHANGES_REQUESTED** [review]
Some fixes needed

File: tolerance-policies/apps/monitor/src/test/java/com/redhat/training/NativeInfoResourceIT.java (hunk 1 above)
Comment (rruizher) [reply tolerance-policies/apps/monitor/src/test/java/com/redhat/training/NativeInfoResourceIT.java]: Done

File: tolerance-policies/apps/monitor/src/main/java/com/redhat/training/MonitorResource.java
Comment (rruizher) [reply tolerance-policies/apps/monitor/src/main/java/com/redhat/training/MonitorResource.java:1]: Done, sorry

**Review by someth2say — APPROVED** [review]

**Pull request merged**